components/sticky/content/src/index.js

```diff
@@ -0,0 +1,130 @@
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+import throttle from 'lodash.throttle'
+import cx from 'classnames'
+
+const STICKY_CONTENT_POSITION_CLASSNAME = {
+  top: 'sui-StickyContent--top',
+  bottom: 'sui-StickyContent--bottom'
+}
+
+const SCROLL_EVENT_LISTENER_OPTIONS = {
+  capture: true,
+  passive: true
+}
+
+const THROTTLE_TIME = 250
+
+const NO_OP = () => {}
+
+export default class StickyContent extends Component {
+  static displayName = 'StickyContent'
+
+  static propTypes = {
+    /**
+     * Component's children node.
+     */
+    children: PropTypes.node.isRequired,
+    /**
+     * Flag to indicate that wrapped node will have "sticky" behavior: position relative until is out of view when scrolling, and then fixed to the defined position.
+     */
+    sticky: PropTypes.bool,
+    /**
+     * Flag to indicate that wrapped node will be directly "fixed" to the defined position.
+     */
+    fixed: PropTypes.bool,
+    /**
+     * Modifier: indicates where the component will be fixed ('top' or 'bottom').
+     */
+    position: PropTypes.oneOf(Object.keys(STICKY_CONTENT_POSITION_CLASSNAME)),
+    /**
+     * Modifier: full width (100%).
+     */
+    fullWidth: PropTypes.bool,
+    /**
+     * DOM Element SELECTOR that we will use to measure its scrollTop property to calculate the total scrolled distance in the page. Normally, we would use the document.documentElement always,
+     * but in sui-studio, regarding to its layout position and overflow styling, we have to measure the document.body.scrollTop instead.
+     */
+    scrollableElementSelector: PropTypes.string,
+    /**
+     * Callback to execute when sticky=true and the wrapper position style attribute changes between "relative" and "fixed".
+     */
+    onDisplayChange: PropTypes.func
+  }
+
+  static defaultProps = {
+    position: Object.keys(STICKY_CONTENT_POSITION_CLASSNAME)[0],
```

Comment:
I would use STICKY_CONTENT_POSITION_CLASSNAME.top directly. Changing the order or adding a new sticky classname could change the default behavior without notice.